Tests/MongoSwiftSyncTests/MongoCursorTests.swift

```diff
@@ -56,6 +56,113 @@ final class MongoCursorTests: MongoSwiftTestCase {
         }
     }
 
```

Comment:
the diff is being calculated/displayed weirdly, but all I did in this file was create a new test class and move the tests using tailable cursors down into it.